client/components/NetworkList.vue

```diff
@@ -123,6 +223,110 @@ export default {
 				order: channel.network.channels.map((c) => c.id),
 			});
 		},
+		toggleSearch(event) {
+			event.preventDefault();
+
+			if (this.$refs.searchInput === document.activeElement) {
+				this.closeSearch();
+				return;
+			}
+
+			this.activateSearch();
+		},
+		activateSearch() {
+			if (this.$refs.searchInput === document.activeElement) {
+				return;
+			}
+
+			this.sidebarWasOpen = this.$store.state.sidebarOpen;
```

Comment:
This is only set when using the keybind, not when simply focusing in already open sidebar.

Perhaps instead set this to true to mean "sidebar was closed, and we opened it just now", that way it's simple to track correctly.

Reply:
So only set it to true if `this.$store.state.sidebarOpen` is false?

Reply:
Should be.

Reply:
This should be fixed now.